src/demo-app/demo-app.ts

```diff
@@ -1,12 +1,33 @@
 import {Component} from 'angular2/core';
 import {MdButton} from '../components/button/button';
-
+import {MD_CARD_DIRECTIVES} from '../components/card/card';
 
 @Component({
   selector: 'demo-app',
   providers: [],
   templateUrl: 'demo-app/demo-app.html',
-  directives: [MdButton],
+  styles: [`
```

Comment:
This can go in a follow-up PR, but could you move these styles to a separate file?